sopel/cli/run.py

```diff
@@ -131,7 +131,10 @@ def add_legacy_options(parser):
                             "use `sopel restart` instead)"))
     parser.add_argument("-l", '--list', action="store_true",
                         dest="list_configs",
-                        help="List all config files found")
+                        help=(
+                            "List all config files found"
+                            "(deprecated, and will be removed in Sopel 8; "
+                            "use `sopel-config list` instead)"))
```

Comment:
Someone remind me to add this in the v7 migration guide's CLI overhaul section.

Reply:
Ping. :)

Reply:
Double-backtick note still applies to latest PR revision.